Expecto/Expecto.fs

```diff
@@ -615,14 +614,18 @@ module Impl =
         summary = fun _ summary ->
           let spirit =
             if summary.successful then
+#if !NETSTANDARD1_6
```

Comment:
I don't see why this is commented out. According to [this issue](https://github.com/dotnet/corefx/issues/10054) there's an API to check console output encoding.